### ISSUE TITLE
Assertion failed in align.c while running f5c eventalign on Nanopore Direct RNAseq

### ISSUE THREAD

**Issue by Euchiz** [hasindu2008/f5c#177]
Hello,

I encountered an error while running f5c eventalign on my Nanopore Direct RNAseq data. The error message is as follows:
```
f5c: src/align.c:457: int32_t align(AlignedPair*, char*, int32_t, event_table, model_t*, uint32_t, scalings_t, float): Assertion `outIndex < (int)(n_events * 2)' failed.
```
The command line that I used was:
```
f5c eventalign -t 64 --pore rna004 --rna -b input.bam -g gencode.v46.transcripts.fa -r input.fastq --slow5 input.blow5 \
    --signal-index --scale-events
```
#### Details:

- **BAM Input**: Output from `dorado-v0.7.2`, aligned to GENCODEv46 transcripts by `minimap2`.
- **BLOW5 Input**: Converted from raw POD5 (rna004) files and merged using `bluecrab` and `slow5tools`.
- **f5c Version**: Latest build from release/binaries.
- **Error**: The error occurred after generating approximately 300GB of events. There are no other error messages or read ID provided except the assertion failure mentioned above.
  
Could you please advise on what might be causing this error and how I can resolve it? 

Thank you for your assistance!

**Comment by hasindu2008** [comment]
Hello,

This rare corner case that seems to happen with rna004 was fixed a couple of months ago and is still in the dev branch. Would you be able to compile the dev branch and try if it works for you? 

Regards

**Comment by Euchiz** [comment]
Thank you for your prompt response!

I cloned the development branch as you suggested, compiled it, and reran the eventalign function on my Nanopore direct RNA-seq data. I am happy to report that the issue is now resolved, and everything worked smoothly!

**Comment by hasindu2008** [comment]
Great, I have released v1.5 that incorporates this fix.
